src/compose/app.ts

```diff
@@ -355,9 +355,7 @@ export class App {
 			// check that we want to stop it, and that it isn't stopped
 			return (
 				serviceTarget.config.running === false &&
-				// When we issue a stop step, we remove the containerId from this structure.
-				// We check if the container has been removed first, so that we can ensure we're not providing multiple stop steps.
-				applicationManager.containerStarted[serviceCurrent.containerId!] == null
+				serviceCurrent.status !== 'Stopped'
```

Comment:
I quickly made this change so might need more thought.

Reply:
That sounds correct, although this will only work for now while we don't have a "stopped" target. If we did, this would cause the supervisor to keep stopping the container, despite the user starting it through the command line or API, for instance.

The correct way to have stopped targets will be to use `finishedAt` 

Just leaving the comment for posterity (although we could add a comment in the code).